_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
+Gaia. Transactional metadata is stored on the Blockstack blockchain and user
+application data is stored in Gaia hub. Storing data off of the blockchain
+ensures that Blockstack applications can provide users with high-performance and
+high availability for data reads and writes without introducing central trust
+parties.
+
+* TOC
+{:toc}
+
+##  Understand Gaia in the Blockstack architecture
+
+The following diagram depicts the Blockstack architecture and Gaia's place in it:
+
+{% include architecture.md %}
+
+## User control or how is Gaia decentralized?
+
+Within the provider's storage system a Gaia hub runs as a service. This service
```

Comment:
Similar to my note above, I'm not sure what a storage system is, let alone a "provider's" storage system.

Reply:
This sentence that starts with "This service authenticates writes to a location" doesn't make a lot of sense to me, can you try rephrasing?

Reply:
I've tweaked the language on the architecture section in light of this and your previous comments.